test/e2e/scheduler_predicates.go

```diff
@@ -1213,8 +1211,7 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 
 		By("Trying to launch a pod that with PodAffinity & PodAntiAffinity setting as embedded JSON string in the annotation value.")
 		labelPodName := "with-newlabels"
-		nodeSelectionRoot := filepath.Join(framework.TestContext.RepoRoot, "test/e2e/testing-manifests/node-selection")
-		testPodPath := filepath.Join(nodeSelectionRoot, "pod-with-pod-affinity.yaml")
+		testPodPath := string(framework.ReadOrDie("test/e2e/testing-manifests/node-selection/pos-with-node-affinity.yaml"))
```

Comment:
I just sent a PR #30061 to change it back, feel free to close if you have better solution :)